### PR TITLE
[zh]Update content/zh/docs/concepts/storage/volumes.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -2102,12 +2102,39 @@ for more information.
...
+operations like scanning of disk devices and mounting of file systems. These operations
+differ for each host operating system. For Linux worker nodes, containerized CSI node
+node plugins are typically deployed as privileged containers. For Windows worker nodes,
+privileged operations for containerized CSI node plugins is supported using
+[csi-proxy](https://github.com/kubernetes-csi/csi-proxy), a community-managed,
+stand-alone binary that needs to be pre-installed on each Windows node.
+-->
+CSI 节点插件需要执行各种特权，
+比如扫描磁盘设备和挂载文件系统之类的操作。
+这些操作在每个主机操作系统都不同。 对于 Linux 工作节点，CSI 节点插件通常部署为特权容器。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:2128]:
```suggestion
这些操作因主机操作系统而异。对于 Linux 工作节点，CSI 节点插件通常部署为特权容器。
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -2102,12 +2102,39 @@ for more information.
...
+对于 Windows 工作节点，CSI 节点插件的特权操作支持使用社区管理的
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:2129]:
```suggestion
对于 Windows 工作节点，CSI 节点插件的特权操作使用社区管理的
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 3]
```diff
@@ -2102,12 +2102,39 @@ for more information.
...
+[csi-proxy](https://github.com/kubernetes-csi/csi-proxy)，
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:2130]:
```suggestion
[csi-proxy](https://github.com/kubernetes-csi/csi-proxy) 提供支持，
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 4]
```diff
@@ -2102,12 +2102,39 @@ for more information.
...
+需要预先安装在每个 Windows 节点上的独立二进制文件。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:2131]:
```suggestion
需要预先在每个 Windows 节点上单独安装二进制文件。
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 5]
```diff
@@ -2156,6 +2194,16 @@ FlexVolume 是一个使用基于 exec 的模型来与驱动程序对接的树外
 Pod 通过 `flexvolume` 树内插件与 FlexVolume 驱动程序交互。
 更多详情请参考 FlexVolume [README](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md#readme) 文档。
 
+<!--
+The following FlexVolume [plugins](https://github.com/Microsoft/K8s-Storage-Plugins/tree/master/flexvolume/windows),
+deployed as PowerShell scripts on the host, support Windows nodes:
+-->
+以下 FlexVolume [插件](https://github.com/Microsoft/K8s-Storage-Plugins/tree/master/flexvolume/windows)，
+在主机上使用 PowerShell 脚本部署，支持 Windows 节点：
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:2202]:
```suggestion
在主机上使用 PowerShell 脚本部署的如下 FlexVolume
[插件](https://github.com/Microsoft/K8s-Storage-Plugins/tree/master/flexvolume/windows)
支持 Windows 节点：
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 6]
```diff
@@ -2102,12 +2102,39 @@ for more information.
...
-
+For more information on how to develop a CSI driver, refer to the
+[kubernetes-csi documentation](https://kubernetes-csi.github.io/docs/)
 -->
 有关如何开发 CSI 驱动的更多信息，请参考 [kubernetes-csi 文档](https://kubernetes-csi.github.io/docs/)。
 
+<!--
+#### Windows CSI proxy
+-->
+#### Windows CSI 代理
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:2113]:
```suggestion
#### Windows CSI 代理   {#windows-csi-proxy}
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 7]
```diff
@@ -2102,12 +2102,39 @@ for more information.
...
+<!--
+CSI node plugins need to perform various privileged
+operations like scanning of disk devices and mounting of file systems. These operations
+differ for each host operating system. For Linux worker nodes, containerized CSI node
+node plugins are typically deployed as privileged containers. For Windows worker nodes,
+privileged operations for containerized CSI node plugins is supported using
+[csi-proxy](https://github.com/kubernetes-csi/csi-proxy), a community-managed,
+stand-alone binary that needs to be pre-installed on each Windows node.
+-->
+CSI 节点插件需要执行各种特权，
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volumes.md:2126]:
```suggestion
CSI 节点插件需要执行多种特权操作，
```

File: content/zh/docs/concepts/storage/volumes.md [hunk 8]
```diff
@@ -2137,6 +2164,17 @@ are listed in [Types of Volumes](#volume-types).
 上面的[卷类型](#volume-types)节列出了支持 `CSIMigration` 并已实现相应 CSI
 驱动程序的树内插件。
 
+<!--
+The following in-tree plugins support persistent storage on Windows nodes:
+-->
+Windows 节点上的持久存储支持以下 in-tree 插件 ：
```
Comment (ydFu) [inline content/zh/docs/concepts/storage/volumes.md:2170]:
```suggestion
Windows 节点上的持久存储支持以下树内插件（In-Tree Plugin）：
```
[refer to ](/zh/docs/concepts/storage/volumes/)

**Comment by tengqm** [comment]
@kinzhi Please rebase this one onto 'main' branch.

**Comment by tengqm** [comment]
@kinzhi Still working on this?

**Comment by ydFu** [comment]
@kinzhi ,is there anything to need help?

**Pull request closed without merging**